web-registry/src/lib/ecocredit.ts

```diff
@@ -346,3 +358,133 @@ export const queryEcoClassInfo = async (
     throw new Error(`Error fetching class info: ${err}`);
   }
 };
+
+/**
+ *
+ * REGEN LEDGER API QUERIES
+ *
+ */
```

Comment:
I don't think this is the best place for having those queries
it looks weird to have some `bank` related queries in a file that's called `ecocredit.ts`

Reply:
Do you think it's okay if it's something we take into consideration and improve in future iterations/tasks?

Reply:
I believe that wouldn't be to much work to move these to another file...

Reply:
Yeah, I'd appreciate a separate `bank.ts` file. That way we keep following the cosmos module pattern.

Reply:
Done! I have moved the functions to their corresponding files and default location `src/lib/bank.ts` and `src/lib/basket.ts`. 

I would suggest taking this further and creating a ledger module that encapsulates all of the ledger-related logic, as I quickly showed the other day at the architecture meeting. Something like this:

`src/ledger/`

![image](https://user-images.githubusercontent.com/5287737/164112553-c9659b6f-e857-4e4b-adc1-65fbf2c7247d.png)

Reply:
unresolving, this file needs to be cleaned up or you just forgot to push your changes...

Reply:
the changes were uploaded, but I hadn't removed the code from the previous version (rest api). Now I have already cleaned the ecocredit files in /types and /lib. do you mean that?